recipe/meta.yaml

```diff
@@ -50,6 +50,10 @@ source:
       # qtscript
       - patches/0001-qtscript-mark-cti_vm_throw-as-REFERENCED_FROM_ASM.patch
 
+      # OSX BigSur compatibility
+      # Remove on next release of 5.12.11
+      - patches/0100-osx_bigsur_compatibility.diff
+
```

Comment:
Please remove this. It's not needed as we are using 10.15 SDK

Reply:
I should have elaborated on this a little more. And I definitely should have kept the link to the original PR to the qt source code.

This adds checks for BigSur which always enables layering. Otherwise, applications need to specify an environment variable will simply get a blank window https://github.com/napari/napari/issues/1393